src/finput.js

```diff
@@ -77,6 +77,9 @@ class Finput {
   get options() {
     return this._options;
   }
+  set options(options) {
+    this._options = options;
```

Comment:
Good spot. I was already using Object.assign in the factory function lower down to avoid mutations via references. I'll switch to spread syntax, supplement existing options and move to here so that we're only exposing functionality in the factory function.